backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/HandoverMailService.java

```diff
@@ -76,7 +81,10 @@ public HandoverMail handOver(
       DocumentationUnit documentationUnit, String receiverAddress, String issuerAddress) {
     XmlTransformationResult xml;
     try {
-      xml = xmlExporter.transformToXml(getTestDocumentationUnit(documentationUnit));
+      xml =
+          xmlExporter.transformToXml(
+              getTestDocumentationUnit(
+                  textCheckService.addNoIndexTagsForPublication(documentationUnit)));
```

Comment:
I'd rather not call this `addNoIndexTagsForPublication` but `addNoIndexTagsForhandOver`